scripts/exgdas_global_atmos_ensanal_run.sh

```diff
@@ -119,7 +119,7 @@ for fieldset in $fieldsets; do
 done
 
 # CRTM coeffs
-$NLN $FV3JEDI_FIX/crtm/2.3.0_jedi $DATA/crtm
+${NLN} "${FV3JEDI_FIX}"/crtm/2.3.0 "${DATA}"/crtm
```

Comment:
Same as above.

Reply:
Requested change made and committed at 5ecf08d0.